src/main/java/net/coderbot/iris/rendertarget/FramebufferBlitter.java

```diff
@@ -27,7 +27,7 @@ public static void copyDepthBufferContent(GlFramebuffer from, RenderTarget to) {
 		int height = to.height;
 
 		// TODO: Support OpenGL 2.1?
-		GL30C.glBlitFramebuffer(0, 0, width, height, 0, 0, width, height,
+		IrisRenderSystem.blitFramebuffer(0, 0, width, height, 0, 0, width, height,
```

Comment:
What's the difference between _glBlitFramebuffer and blitFramebuffer here?

Reply:
IIRC _glBlitFramebuffer didn't support the arguments we used